spec/50-privacy.md

```diff
@@ -14,8 +14,10 @@ Note, both privacy concerns of `traceparent` field are theoretical rather than p
 
 ## Privacy of tracestate field
 
-The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific trace-identification information across different distributed tracing systems. 
+The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific trace-identification information across different distributed tracing systems.
 
 Tracing systems MUST NOT include any personally identifiable information in the `tracestate` header.
 
 Platforms and tracing systems extremely sensitive to personal information exposure MAY implement selective removal of values corresponding to the unknown keys. This mutation of the `tracestate` field is not forbidden, but highly discouraged. As it defeats the purpose of this field for allowing multiple tracing systems to collaborate.
+
+*NOTE*: Should we restart the trace in this situation?
```

Comment:
please remove the note if this is not work in progress

Reply:
Can I take care of leftover notes when all changes have landed?
They will be gone by the time we give the 'go' for CR.

Reply:
my understanding is that this PR was intended as 

>It should only contain editorial changes.

So let's not introduce unfinished sentences and notes

Reply:
I think that a note does not change the spec in any way but tells that there is something open to discuss. As such it is as editorial as it can get because it is not even intended to still be there in CR.
Regardless of that, I removed the note.

Reply:
I agree with Daniel. We should be clear about areas where we have open discussions and where behavior is not fully defined. We owe this to people implementing the spec. Ideally, we also link to an issue or PR.

Every other behavior would be unprofessional with regards to openness. If a potential implementor reads this and has a strong opinion he will also feel more motivated to provide feedback.

Reply:
If we have cases where we undecided like this - I'd suggest to put a bit more formal text like "Note, this specification is not defined exact recommendation of tracing system or platform behavior in this situation. Removal of `tracestate` entries may be harmful for interoperability and future versions of this specification may enforce restarting `traceparent` in these cases.". This will explains what comments says better.

As for this particular comment - spec allows to modify `tracestate` entries. So here depending on implementation of privacy-enforcement proxy or library - it may decide the behavior on its own. I can see how organization may have an outgoing traffic proxy that clears out internal-use-only `tracestate` entries. Forcing to restart in this situation will be damaging for interoperability

Reply:
This is why I wanted to link to an issue. People can then follow the discussion thread and we will not have to revisit the same discussion over again. Does this work for you?